libraries/chain/controller.cpp

```diff
@@ -304,6 +315,8 @@ struct controller_impl {
     read_mode( cfg.read_mode ),
     thread_pool( "chain", cfg.thread_pool_size )
    {
+
+      // reset the
```

Comment:
Looks like this was added unintentionally and also the newline below.

Reply:
Gotcha